index.html

```diff
@@ -839,33 +841,33 @@
               <var>acceptPromise</var> with that error and terminate this
               algorithm.
               </li>
-              <li>Determine which <a>payment handlers</a> support
-              <var>identifier</var>. For each resulting payment handler, if
-              payment method specific capabilities supplied by the payment
-              handler match those provided by <var>data</var>, the payment
-              handler matches.
+              <li>Add each <a>payment handler</a> that supports
```

Comment:
We discussed this on today's Editor's call. @zkoch gave me a "reality check": when Chrome does work in concert with third-party payment methods (native apps in this case), they do "fail fast" at this point. Given that what I sought was confirmation that another use agent was in fact doing doing the filtering, then I concur and will add the text that @ianbjacobs suggested.